### PR TITLE
Add vessel tests.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: tests/vessels/test_generator.toit [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import host.file
+
+DATA_256 := ((List 256: it).map: "$(%3d it), ").join ""
+
+main args:
+  out_file := args[0]
+  size := int.parse args[1]
+
+  data := "#["
```
Comment (kasperl) [inline tests/vessels/test_generator.toit:13]: Add a comment that explains what this does and why. It isn't super easy to grok what it builds and does.

File: tests/vessels/test_generator.toit [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2022 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+import host.file
+
+DATA_256 := ((List 256: it).map: "$(%3d it), ").join ""
```
Comment (kasperl) [inline tests/vessels/test_generator.toit:7]: Add a comment that shows the shape of the string, so every reader do not have to run this through their head.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tests/vessels/test_generator.toit (hunk 2 above)
Comment (floitsch) [reply tests/vessels/test_generator.toit:17]: done.

File: tests/vessels/test_generator.toit (hunk 1 above)
Comment (floitsch) [reply tests/vessels/test_generator.toit:27]: sorry. should have done this without your request.
done.

**Pull request merged**